jobs/build/build-sync/Jenkinsfile

```
@@ -110,89 +109,152 @@ node {
+            currentBuild.displayName += " [EXCLUDE ${excludeArches.join(', ')}]"
+            if ( !arches.containsAll(excludeArches) )
+                error("Trying to exclude arch ${excludeArches} not present in known arches ${arches}")
+            arches.removeAll(excludeArches)
+        }
+        currentBuild.description = "Arches: ${arches.join(', ')}"
+
+        if ( params.DEBUG ) {
+            logLevel = " --loglevel=5 "
+        }
+
+        imageList = commonlib.cleanCommaList(params.IMAGES)
+        if ( imageList ) {
+            echo("Only syncing specified images: ${imageList}")
+            currentBuild.description += "<br>Images: ${imageList}"
+        }
+
+        failCountFile = "${BUILD_VERSION}-assembly.${ASSEMBLY}.count"
+    }
 
     stage("Version dumps") {
         buildlib.doozer "--version"
         echo "${buildlib.DOOZER_BIN}"
         sh "oc version -o yaml"
     }
 
-    failCountFile = "${BUILD_VERSION}-assembly.${ASSEMBLY}.count"
-    currentBuild.displayName = "${BUILD_VERSION} - ${ASSEMBLY}"
-
-    try {
-
+    stage ("build sync") {
+        sh "rm -rf ./artcd_working && mkdir -p ./artcd_working"
+        def cmd = [
+            "artcd",
+            "-v",
+            "--working-dir=./artcd_working",
+            "--config=./config/artcd.toml",
+        ]
+        if (params.DRY_RUN) {
+            cmd << "--dry-run"
+        }
+        cmd += [
+            "build-sync",
+            "--version=${params.BUILD_VERSION}",
+            "--assembly=${params.ASSEMBLY}"
+        ]
+        if (params.PUBLISH) {
+            cmd << "--publish"
+        }
+        cmd << "--data-path=${params.DOOZER_DATA_PATH}"
+        if (params.EMERGENCY_IGNORE_ISSUES) {
+            cmd << "--emergency-ignore-issues"
+        }
+        if (params.EMERGENCY_IGNORE_ISSUES) {
+            cmd << "--emergency-ignore-issues"
+        }
         if (params.RETRIGGER_CURRENT_NIGHTLY) {
-            if (params.ASSEMBLY != "stream") {
-                error "Cannot use with assembly other than stream. Exiting."
-            }
-            if (params.DRY_RUN) {
-                echo "Would have triggered new release cut in release controller."
-            } else {
-                echo "Triggering release controller to cut new release using previously synced builds..."
-                buildlib.oc("--kubeconfig ${buildlib.ciKubeconfig} -n ocp tag registry.access.redhat.com/ubi8 ${params.BUILD_VERSION}-art-latest:trigger-release-controller")
-                echo "Sleeping so that release controller has time to react..."
-                sleep(60)
-                buildlib.oc("--kubeconfig ${buildlib.ciKubeconfig} -n ocp tag ${params.BUILD_VERSION}-art-latest:trigger-release-controller -d")
-            }
-            return
+            cmd << "--retrigger-current-nightly"
+        }
+        if (params.DOOZER_DATA_GITREF) {
+            cmd << "--doozer-data-gitref=${params.DOOZER_DATA_GITREF}"
```

Since we call ocp-build-data repo url `--data-path`, maybe it is better to name this parameter without doozer prefix:
```suggestion
            cmd << "--data-gitref=${params.DOOZER_DATA_GITREF}"
```

---

Fixed in f5103418c28f538d767eeff6e69722001d25c2e8